src/lib/components/lines-and-dots/workflow-error.svelte

```diff
@@ -78,28 +80,12 @@
           relative: $relativeTime,
         })}
       </div>
-      <div class="flex flex-col gap-2 bg-space-black p-4 text-white">
-        <p>{translate('common.failure')}</p>
-        <CodeBlock
-          content={error.attributes?.failure?.message || ''}
-          language="text"
-          copyIconTitle={translate('common.copy-icon-title')}
-          copySuccessIconTitle={translate('common.copy-success-icon-title')}
-        />
-        <p>{translate('common.stack-trace')}</p>
-        <CodeBlock
-          content={error.attributes?.failure?.stackTrace || ''}
-          language="text"
-          copyIconTitle={translate('common.copy-icon-title')}
-          copySuccessIconTitle={translate('common.copy-success-icon-title')}
-        />
-        <p>{translate('common.source')}</p>
-        <CodeBlock
-          content={error.attributes?.failure?.source || ''}
-          language="text"
-          copyIconTitle={translate('common.copy-icon-title')}
-          copySuccessIconTitle={translate('common.copy-success-icon-title')}
-        />
+      <div class="flex flex-col gap-2 bg-primary p-4">
+        {#if error.attributes?.failure}
+          <AccordionGroup>
+            <WorkflowErrorStackTrace failure={error.attributes.failure} />
+          </AccordionGroup>
```

Comment:
Is it possible to have more than one failure here? Looking at the screenshots there are multiple, but looking at the code it looks like we'd only every have one `Accordion` in which case we probably don't need an `AccordionGroup` here 

```suggestion
          <WorkflowErrorStackTrace failure={error.attributes.failure} />
```

Reply:
If you look in the WorkflowErrorStackTrace, it recursively calls itself. These failures can be deeply nested (apparently how some SDKs work like .NET and Java). So it can have one or many of the accordions.